BT3103AppDev1/final-project-l1-final-project-group-12
# Ceated Home landing page

## Description
In this pull request, I have introduced several visual improvements to the Smartfolio's home page interface. The changes touch upon both the structure of the content and the styling to enhance the user experience.

## Related Issue
To solve the issue: [3](https://github.com/BT3103AppDev1/final-project-l1-final-project-group-12/issues/3)

## Motivation and Context

The motivation behind this change is to provide users with an interface to understand Smartfolio. Users can now easily learn more about Smartfolio and navigate to different parts of the app.

## How Has This Been Tested?



I did the following to test the code:
- Load the page on multiple screen sizes to ensure responsiveness.
- Hover over the Sign up now! button to check the underline effect.
- Verify that all images load correctly and are properly aligned.
- Check the text for any typographical errors and ensure it's legible against the backgrounds.

## Screenshots (if appropriate):
![image](https://github.com/BT3103AppDev1/final-project-l1-final-project-group-12/assets/92180655/b4196bef-3b18-471d-ae72-5334be30984e)
*note: pagebreak is due to save to pdf limitations

## Types of changes


## Checklist:
